spec/javascripts/LoginPage/Login3scaleForm.spec.jsx

```diff
@@ -27,18 +27,62 @@ it('should render HiddenInputs component', () => {
   expect(wrapper.find(HiddenInputs).exists()).toEqual(true)
 })
 
-it('should set username state', () => {
-  const wrapper = mount(<Login3scaleForm {...props}/>)
-  jest.spyOn(wrapper.instance(), 'handleTextInputUsername')
-  wrapper.instance().handleTextInputUsername('foo')
-  expect(wrapper.instance().handleTextInputUsername).toHaveBeenCalled()
-  expect(wrapper.state().username).toEqual('foo')
+describe('username', () => {
+  it('should set username and validation state to true', () => {
+    const event = {
+      currentTarget: {
+        name: 'username',
+        value: 'Bob',
+        type: 'text'
+      }
+    }
+    const wrapper = mount(<Login3scaleForm {...props}/>)
```

Comment:
Would it be possible to use `shallow` instead? Using `mount` is a bit overkilling.

Reply:
Nope, because all `wrapper.find(input)` fails